pkg/operator/mon/mon_test.go

```diff
@@ -465,3 +466,44 @@ func TestHostNetworkPortIncrease(t *testing.T) {
 	sEndpoint = strings.Split(c.clusterInfo.Monitors["mon2"].Endpoint, ":")
 	assert.Equal(t, strconv.Itoa(cephmon.DefaultPort+1), sEndpoint[1])
 }
+
+func TestCheckHealthNotFound(t *testing.T) {
```

Comment:
looks like this unit test failed in the build

Reply:
I fixed the test, it was caused by the actual endpoints "changing" one time being `mon1,mon2` or `mon2,mon1`.